include/libopencm3/stm32/l4/i2c.h

```diff
@@ -0,0 +1,48 @@
+/** @defgroup i2c_defines I2C Defines
+
+@brief <b>Defined Constants and Types for the STM32L4xx I2C </b>
+
+@ingroup STM32L4xx_defines
+
+@version 1.0.0
+
+@date 12 October 2012
+
+LGPL License Terms @ref lgpl_license
+ */
+
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * This library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef LIBOPENCM3_I2C_H
+#define LIBOPENCM3_I2C_H
+
+#include <libopencm3/stm32/common/i2c_common_v2.h>
+
+/**@{*/
+
+#define I2C3				I2C3_BASE
+
+/* for API compatibility */
+#define RCC_APB1RSTR			RCC_APB1RSTR1
+#define RCC_APB1RSTR_I2C2RST		RCC_APB1RSTR1_I2C2RST
+#define RCC_APB1RSTR_I2C1RST		RCC_APB1RSTR1_I2C2RST
```

Comment:
It was for i2c_reset only. It is not needed when rcc_periph_reset_pulse is used. Will be removed.